pkg/sql/logictest/testdata/logic_test/inverted_index

```diff
@@ -269,3 +266,27 @@ SELECT * from d where b @> '{}' ORDER BY a;
 
 statement error pq: constraints = , table ID = 53, index ID = 2: can't look up empty JSON
 SELECT * from d where b @> '[]' ORDER BY a;
+
+statement ok
+INSERT INTO d VALUES (20, '["a", "a"]')
+
+query IT
+SELECT * from d where b @> '["a"]' ORDER BY a;
+----
+20  ["a","a"]
+
+statement ok
+INSERT INTO d VALUES (21, '[{"a":"a"}, {"a":"a"}]')
+
+query IT
+SELECT * from d where b @> '[{"a":"a"}]' ORDER BY a;
+----
+21  [{"a":"a"},{"a":"a"}]
+
+statement ok
+INSERT INTO d VALUES (22,  '[[[["a"]]],[[["a"]]]]')
+
+query IT
+SELECT * from d where b @> '[[[["a"]]]]' ORDER BY a;
+----
+22  [[[["a"]]],[[["a"]]]]
```

Comment:
It's quite surprising that code in util/json is being tested using logictests. Besides these logic tests are you able to write tests within the util/json package for this change? Thanks!